Additional instructions & added screenshots (#210)
ServiceInnovationLab/serviceinnovationlab.github.io: pages/gettingStarted.md

## Getting Started

***NOTE:** These steps only apply to people involved with the Lab. For the first time adding content & getting set-up, it will take longer than usual.*

### Create a GitHub account

- _Go to_ [GitHub](https://github.com/)
- _Click on_ **Sign up** (& follow the steps). **Note:** _Choose_ the Free Subscription.
- Remember to _verify_ your email address (in order to start using GitHub's features).

### Set up 2FA

- Go to **Settings** (icon on top right-hand corner of page) -> **Security** -> _scroll to_ **Two-factor authentication**,  _choose_ the appropriate method for you (& follow the steps).
- _Download_ your recovery codes so that you can recover your account if you lose your 2FA credentials.
- Once done, _contact_ one of the Github repository admins & they will add you to become a member of the repository. Only if you are approved as a member, then you add content to the repository. When approved, _go to_ your email and accept the invite.

## Adding content to the team page

- _Navigate to_ the [Toolkit](https://github.com/ServiceInnovationLab/serviceinnovationlab.github.io)'s repository.

> https://github.com/ServiceInnovationLab/serviceinnovationlab.github.io

Reviewer: ```suggestion
- _Navigate to_ the [Toolkit](https://github.com/ServiceInnovationLab/serviceinnovationlab.github.io)'s repository.
<https://github.com/ServiceInnovationLab/serviceinnovationlab.github.io>
```

- _Click on_ ![2FA](/assets/img/screenshot-instructions/2fa.png)
- _Download_ & _save_ your recovery codes so that you can recover your account if you lose your 2FA credentials.
- You'll then be re-directed to the next page where depending on the authentication method you have chosen you'll either need to fill in your mobile number (for sms) **OR** needing to scan the QR code (for app). Either way, once you receive the code after following the steps, enter it in the section (as shown below) & _click_ **Enable** when done:![Enable 2FA](/assets/img/screenshot-instructions/enable-2fa.png)

> - _Contact_ one of the `Service Innovation Lab's` repository admins (ie. `Brenda` or `Jacob`) & they will add you to become a member of the repository. You'll be sent an email when approved. _Go to_ your email and accept the invite so that you can add to the repository.

Reviewer: ```suggestion
- _Contact_ one of the `Service Innovation Lab's` repository admins (ie. `Brenda` or `Jacob`) & they will add you to the team on github. You'll receive an invitation by email. _Go to_ your email and accept the invitation.
```
